docs/example_workflow_explained.rst

```diff
@@ -0,0 +1,316 @@
+        'bmorph_overlap': overlap,
+        'condition_var': condition_var
+    }
+    
+    univariate_config = {
+        'train_window': train_window,
+        'bmorph_window': bmorph_window,
+        'reference_window': reference_window,
+        'bmorph_interval': interval,
+        'bmorph_overlap': overlap,
+    }
+
+You made it! Now we can actually bias correction with ``bmorph``! Depending
+on the size of your data and use of parallelism or not, the following cells
+will likely take the longest to run, so make certain everything else looks
+good to you before running it.
+
+First off we run the Independent Bias Corrections, which is completely contained
+in the cell below. If you are interested in ``bmorph``'s spatial consitency and conditioing
+bias corrections, this cell is not it. However, it can be useful to run at least once
+so you have a baseline method to compare to as you fine tune variables.
+
+Here we run through each of the gauge sites and correct them 
+individually. Since independent bias correction can only be performed
+at locations with reference data, corrections are only performed at
+the gauge sites here. If you have not changed any naming conventions
+so far, then there is nothing that you need to alter here, it has all already
+been extracted above for your convenience.
+
+.. code:: ipython3
+
+    ibc_u_flows = {}
+    ibc_u_mults = {}
+    ibc_c_flows = {}
+    ibc_c_mults = {}
+    
+    raw_flows = {}
+    ref_flows = {}
+    
+    for site, seg in site_to_seg.items():
+        raw_ts = basin_met_seg.sel(seg=seg)['IRFroutedRunoff'].to_series()
+        train_ts = basin_met_seg.sel(seg=seg)['IRFroutedRunoff'].to_series()
+        obs_ts = basin_met_seg.sel(seg=seg)['up_ref_flow'].to_series()
+        cond_var = basin_met_seg.sel(seg=seg)[f'up_{condition_var}'].to_series()
+        ref_flows[site] = obs_ts
+        raw_flows[site] = raw_ts
+        
+        ## IBC_U (Independent Bias Correction: Univariate)
+        ibc_u_flows[site], ibc_u_mults[site] = bmorph.workflows.apply_interval_bmorph(
+            raw_ts, train_ts, obs_ts, train_window, bmorph_window, reference_window, interval, overlap)
+        
+        ## IBC_C (Independent Bias Correction: Conditioned)
+        ibc_c_flows[site], ibc_c_mults[site] = bmorph.workflows.apply_interval_bmorph(
+            raw_ts, train_ts, obs_ts, train_window, bmorph_window, reference_window, interval, overlap,
+            raw_y=cond_var, train_y=cond_var, obs_y=cond_var)
+
+
+    
+Here you specify where ``mizuroute`` is installed on your system
+and set up some variables to store total flows.
+
+``region`` will be used to write and load files according to the
+basin's name, make certain to update this with the actual name of
+the basin you are analyzing so you can track where different files
+are writen.
+
+.. code:: ipython3
+
+    mizuroute_exe = # mizuroute designation
+    
+    unconditioned_totals = {}
+    conditioned_totals = {}
+    region = # basin name
```

Comment:
I need to double check the requirement of naming a region, and if this is something we really want to enforce. This is just a note to self.

Reply:
It's currently used to prefix files in case you want to do multiple runs with potentially different data without overriding them. I could change this to `prefix`

Reply:
Yeah, that makes sense. Maybe even something even more descriptive like `output_prefix`